File: Taejin/DS_Module/StackExample.cpp
// stack_stl.cpp
#include <iostream>
#include <stack>

using namespace std;

int main(void) {
	// Delcaration
	stack<int> stck;

	// push
	stck.push( 1 );
	stck.push( 2 );
	stck.push( 3 );

	// top
	cout << stck.top() << endl;

	// pop
	stck.pop();

	return 0;
}

File: Taejin/DS_Module/StackExample.java
// StackExample.java
import java.util.*;

public class StackExample {
	public static void main( String[] args ) {
		// Declaration
		Stack<Integer> stck = new Stack<Integer>();

		// Push values
		stck.push( 1 );
		stck.push( 2 );
		stck.push( 3 );

		// Pop value
		System.out.println( stck.pop() );

		// Top of stack
		System.out.println( stck.peek() );
	}
};
// Reference: https://docs.oracle.com/en/java/javase/13/docs/api/java.base/java/util/Stack.html